test/controllers/TaxRatesAndBandsControllerSpec.scala

```diff
@@ -110,6 +110,25 @@ class TaxRatesAndBandsControllerSpec extends UnitSpec with WithFakeApplication w
 
   "validating the getMaxPersonalAllowance method" when {
 
+    "calling with the year 2021 and with no BPA" should {
+      val result = controller.getMaxPersonalAllowance(2021, None)(fakeRequest)
+
+      "return status 200" in {
+        status(result) shouldBe 200
+      }
+
+      "return a JSON result" in {
+        contentType(result) shouldBe Some("application/json")
+      }
+
+      "return 12300 as the annual exempt amount" in {
```

Comment:
It seems like the spec title `"return 12300 as the annual exempt amount"` should be `"return 12500 as the max personal allowance"`, as `getMaxPersonalAllowance` is the method being called, and the json value is being asserted as `12500` (which is the value of `TaxRatesAndBands20202021.maxPersonalAllowance`). I can see this across the other tests; I don't know if they need changing as well.